backend/src/staticAuth.ts

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+import log = require('./log');
+import _ = require('underscore');
+import md5 = require('md5');
+
+var config = require('../../config/config.js');
+
+interface IStaticAuthUser {
+  name: string;
+  pass: string;
+  groups: string[];
+}
+
+export function authenticate(user: string, password: string, requiredGroup: string, cb: Function) {
+
+  log.info('Trying to authenticate', {user: user});
+
+  var userObject = _.findWhere<IStaticAuthUser, Object>(
+    config.auth.static.users,
+    { name: user, pass: md5(password) }
+    );
```

Comment:
Fix indent (-2 spaces), the closing brace should be on the same level as the line where the opening brace is.